templates/AureliaSpa/ClientApp/boot.ts

```diff
@@ -0,0 +1,15 @@
+import {Aurelia} from 'aurelia-framework';
+import 'bootstrap/dist/css/bootstrap.css';
+import 'bootstrap';
+ 
+// comment out if you don't want a Promise polyfill (remove also from webpack.config.js)
```

Comment:
TBH I'd prefer not to include bluebird in the template, just because the template would be a little simpler without it. People can add their own choice of promise library if they want.